entity_event_generator.tf

```diff
@@ -50,7 +50,7 @@ module "entity_event_lambda_updated_since_query_start_datetime_table" {
   table_name    = local.last_polled_table_name
 }
 
-resource "aws_dynamodb_table_item" "initial_start_date" {
+resource "aws_dynamodb_table_item" "initial_start_date_time" {
```

Comment:
Shouldn't it be`initial_start_datetime`?

Reply:
Ah sorry, I can't read. I've updated it.